astrolabe/runner.py

```diff
@@ -91,29 +93,31 @@ def is_cluster_state(self, goal_state):
         cluster_info = self.cluster_url.get().data
         return cluster_info.stateName.lower() == goal_state.lower()
 
-    def verify_cluster_configuration_matches(self, state):
+    def verify_cluster_configuration_matches(self, expected_configuration):
         """Verify that the cluster config is what we expect it to be (based on
         maintenance status). Raises AssertionError."""
-        state = state.lower()
-        if state not in ("initial", "final"):
-            raise AstrolabeTestCaseError(
-                "State must be either 'initial' or 'final'.")
         cluster_config = self.cluster_url.get().data
         assert_subset(
             cluster_config,
-            self.spec.maintenancePlan[state].clusterConfiguration)
+            expected_configuration.clusterConfiguration)
         process_args = self.cluster_url.processArgs.get().data
         assert_subset(
-            process_args, self.spec.maintenancePlan[state].processArgs)
+            process_args, expected_configuration.processArgs)
 
-    def initialize(self):
+    def initialize(self, no_create=False):
         """
         Initialize a cluster with the configuration required by the test
         specification.
         """
+        
+        self.start_time = _time.time()
```

Comment:
Where is this being used? Also, you should use `astrolabe.utils.Timer` for timing operations - it exposes a simpler interface and uses a monotonic timer to nullify jumps in wall-clock time (which is what is returned by `_time.time`)

Reply:
This was used for the delay to retrieve Atlas logs using the public API but is not currently used for anything, I'll remove.